src/hapi/pipelines/database/operational_presence.py

```diff
@@ -50,17 +48,13 @@ def __init__(
     def populate(self, debug=False):
```

Comment:
Can remove debug parameter, but it would be nice to have a way of seeing how the orgs, org types, and sectors are mapped before deploying so I can easily find values that need to be added to the config/org mapping table

Reply:
For sector and org_type, it is easy because the `data` member variable of each is expanded by each new name that is correctly mapped (but there is no change to the db). I would like to do the same somehow for org and will think about how to do that in the next PR.

Reply:
I added a variable used to OrgInfo can that marks if the mapping is used so the org map can be used to see all the mappings.

Reply:
Could you add a warning message with any unused orgs?

Reply:
I have added a debug option to the command line which will output the org map into a csv if enabled